### ISSUE TITLE
Option to disable page name/title conflict check in PageTables for non-superusers when using external parent page.

### ISSUE THREAD

**Issue by adrianbj** [ryancramerdesign/ProcessWire#1494]
When storing PageTable content pages under a common "external" page, and you create a "page" with the same title as another page, you get the "The name entered is already in use. If you do not modify it, the name will be made unique automatically after you save." error. I think this is just confusing for site editors and not necessary since these page names are not URLs, so it doesn't matter that "-n" is being appended.

Please let me know if I haven't explained this well enough. It is proving to be a real problem on my current project which has just getting client content being created.

Thanks!

**Comment by LostKobrakai** [comment]
I'm not sure this will work. It's true, that name conflicts don't matter for the pagetable field, but those pages are still children of the same parent. This means the name has to be unique to not break things like path selectors ("path=/tablesParent/page-name"). I still think the external option for pagetables is not the best one, but would rather like the option to [hide pages in the tree](https://github.com/ryancramerdesign/ProcessWire/issues/302), so the "pages as children" option does get more viable and less confusing.

**Comment by adrianbj** [comment]
Yes, the name has to be unique, but I don't think the warning needs to exist. I agree there are issue with the external parent page, but as children of the page has problems too - even if hidden (like the dev version of Batch Child Editor now does really well), there is still the problem of them appearing in menus, and being mixed in with children that are meant to be actual child pages.

**Comment by LostKobrakai** [comment]
Ah I thought you were talking about the actual name change itself.

**Comment by adrianbj** [comment]
Ryan - glad you have seen this and tagged it, but I think it is actually quite important. In the setup I have editors are setting up PageTable items (pages) that use common options, eg. Partners, Funders, Collaborators and similar. These are added to several pages on the site so the conflict is coming up quite often and is very confusing. The only option I see is to use the automatic naming option (which I have used in the past), but in general I don't like this if I can avoid it because it makes it hard to visually reference item pages to the parent PageTable field - I hope I explained this well enough.

I guess I just don't understand why the error needs to be reported at all in the scenario of a PageTable. I think it should just go ahead and append the "-n" quietly.
